packages/ucp/docs/getting-started.md

```diff
@@ -0,0 +1,361 @@
+This tutorial is about how to install and use the [@solidlab/ucp](https://github.com/SolidLabResearch/user-managed-access/tree/main/packages/ucp) library utilities.
+
+## Installing
+
+As of today (2 Februari 2024), this package is not on npm.
+Which means installing via `npm install @solidlab/ucp` does **not** work yet.
+
+However, it can still easily installed by adding `"@solidlab/ucp": "https://github.com/woutslabbinck/ucp-enforcement.git"` to your dependencies in package.json.
+
+If no package.json is yet present, you can just create a `package.json` file, copy the contents below and execute `npm i`
+
+```json
+{
+    "dependencies": {
+        "ucp-enforcement": "https://github.com/woutslabbinck/ucp-enforcement.git",
+    }
+}
+```
+
+### Note for installing
+
+Another options currently is to look at [GitPkg](https://gitpkg.now.sh/).
```

Comment:
Integrated above, because I think it's better for people to keep track of the code as it evolves here.

```suggestion
```